benchmarks/functional_autograd_benchmark/utils.py

```diff
@@ -0,0 +1,59 @@
+from collections import defaultdict
+
+# Utilities to make nn.Module functional
+def del_attr(obj, names):
+    if len(names) == 1:
+        delattr(obj, names[0])
+    else:
+        del_attr(getattr(obj, names[0]), names[1:])
+def set_attr(obj, names, val):
```

Comment:
nit: space between `set_attr` and the last line (does flake8 warn about this?)

Reply:
Not sure why it doesn't indeed...
Fixing it.